tests/python-pytest/onnx/test_onnxruntime_cv.py

```diff
@@ -199,8 +199,6 @@ def obj_detection_test_images(tmpdir_factory):
     tmpdir = tmpdir_factory.mktemp("obj_det_data")
     from urllib.parse import urlparse
     test_image_urls = [
-        'https://github.com/apache/incubator-mxnet-ci/raw/master/test-data/images/car.jpg',
-        'https://github.com/apache/incubator-mxnet-ci/raw/master/test-data/images/duck.jpg',
```

Comment:
Why do we remove the images here?

Reply:
Those two images would fail the maskrcnn models. It's not like maskrcnn models do not work on them, but they would output slightly different boxes than mxnet